edfi_api_client/edfi_client.py

```diff
@@ -160,66 +174,6 @@ def get_data_model_version(self) -> Optional[str]:
         else:
             return None
 
-
-    ### Methods related to retrieving the Swagger or attributes retrieved therein
-    def get_swagger(self, component: str = 'resources') -> EdFiSwagger:
-        """
-        OpenAPI Specification describes the entire Ed-Fi API surface in a
-        JSON payload.
-        Can be used to surface available endpoints.
-
-        :param component: Which component's swagger spec should be retrieved?
-        :return: Swagger specification definition, as a dictionary.
-        """
-        swagger_url = util.url_join(
-            self.base_url, 'metadata', self.version_url_string, component, 'swagger.json'
-        )
-
-        payload = requests.get(swagger_url, verify=self.verify_ssl).json()
-        swagger = EdFiSwagger(component, payload)
-
-        # Save the swagger in memory to save time on subsequent calls.
-        self.swaggers[component] = swagger
-        return swagger
-
-    def _set_swagger(self, component: str):
```

Comment:
Remove `_set_swagger()` helper function, since it's use overlaps directly with `get_swagger()`.